src/Persistence.php

```diff
@@ -18,6 +18,19 @@ class Persistence
     use \atk4\core\NameTrait;
     use \atk4\core\DiContainerTrait;
 
+    /** @const string */
+    public const HOOK_INIT_SELECT_QUERY = Persistence\AbstractQuery::HOOK_INIT_SELECT;
```

Comment:
seems like copied - if moved, deprecate with comment on the source side!

Reply:
phpdocs are still pending. QUERY constants were moved to AbstractQuery. This is for BC and will be commented accordingly

Reply:
very easy to forget, always comment before even past!